addOns/retire/src/test/java/org/zaproxy/addon/retire/model/RepoUnitTest.java

```diff
@@ -146,12 +147,4 @@ void shouldReadLibWithAllExtractors() throws IOException {
                         hasEntry("§§version§§", "§§version§§")));
         assertThat(extractors.getUri(), contains("uri 1", "uri 2", "[0-9][0-9a-z._\\-]+?"));
     }
-
-    private static Reader reader(String fileName) throws IOException {
```

Comment:
These changes are not needed, we should have used the constructors from the start...